client/src/components/users/UserCard.vue

```diff
@@ -22,23 +25,38 @@
       <!-- eslint-enable vue/no-mutating-props -->
     </div>
     <div
-      class="card-option"
+      v-if="user.isRevoked()"
+      class="user-revoked"
+    >
+      <user-status-tag :revoked="user.isRevoked()" />
+    </div>
+    <div
+      class="user-card-option"
       v-show="isHovered || menuOpened"
       @click.stop="onOptionsClick($event)"
     >
       <ion-icon :icon="ellipsisHorizontal" />
     </div>
-    <div class="card-content">
-      <ion-avatar class="card-content-avatar">
-        <user-avatar-name
-          class="user-avatar large"
-          :user-avatar="user.humanHandle.label"
-        />
-      </ion-avatar>
-      <ion-text class="user-name body">
-        {{ user.humanHandle.label }}
-      </ion-text>
-      <div class="user-profile">
+    <div class="user-card">
+      <user-avatar-name
+        class="user-card-avatar medium"
+        :user-avatar="user.humanHandle.label"
+      />
+      <div class="user-card-info">
+        <ion-text class="user-card-info__name body">
+          <span>{{ user.humanHandle.label }}</span>
+          <span
+            v-if="mainUser"
+            class="body name-you"
+          >
+            {{ $t('UsersPage.mainUser') }}
```

Comment:
`main user` is weird, it implies that this is a type or more important user, maybe `current user` would make more sense here.

Reply:
Yes, I replaced it.